dateparser/timezone_parser.py

```diff
@@ -44,11 +44,15 @@ def pop_tz_offset_from_string(date_string, as_offset=True):
         return date_string, None
 
 
+def word_is_tz(word):
+    return bool(_search_regex.match(word))
+
+
 def convert_to_local_tz(datetime_obj, datetime_tz_offset):
     return datetime_obj - datetime_tz_offset + local_tz_offset
 
 
-def get_tz_offsets():
+def build_tz_offsets(search_regex_parts):
```

Comment:
@asadurski Should backward-compatibility be a concern here? Otherwise, should we prefix this function with `_`?

Reply:
No, I think the function is hidden so deep - and used only once - that we shouldn't concern ourselves about it.